src/hooks/useFetchChannel.jsx

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const useFetchChannel = (channelId) => {
+  const [channelInfo, setChannelInfo] = useState({});
+
+  useEffect(() => {
```

Comment:
```suggestion
  useEffect(() => {
    if (!channelId) return;
```
현재 코드에서는 만약 `channelId`가 없을 때도 요청이 시도될 수 있을 거 같아요!

`if (!channelId) return;`을 넣어주면 불필요한 API 호출을 막을 수 있을 것 같은데
@faber-h , @devjaeyoon 두 분 의견도 궁금합니다‼️

Reply:
`if (!channelId) return;`을 추가하면 안전한 처리가 될 수 있을 것 같네요!
불필요한 API 호출을 막기 위해 앞으로도 early return 해주는 방식으로 진행하면 좋을 것 같아요~

Reply:
불필요한 API 호출을 막아준다는 점이 좋아요! 좋은 의견 감사드립니다! 해당 내용 반영하겠습니다~